### PR TITLE
Add example data

### PR DESCRIPTION
**Pull request by antony-wilson**
This code provides a `manage.py` option to allow the user to generate some example data for the registry database.

If the registry is running in a virtual environment then ensure that this environment is activated. Also ensure the `DJANGO_SETTINGS_MODULE` environment variable is set. You can then run the command `python manage.py add_example_data`

```
. venv/bin/activate
export DJANGO_SETTINGS_MODULE=drams.vagrant
python manage.py add_example_data
```

By default the script will only run is the registry database is empty. If you wish to add the data to a database that is not empty then use the `--force` option.

```
python manage.py add_example_data --force
```

### CODE REVIEW

**Comment by nathan-cummings-ukaea** [comment]
One issue I have found is that even with a fresh install, an 'admin' user is automatically created, so the check on line 34 of add_external_data.py will always evaluate to True unless the --force option is used. Basically, you have to use the --force option even on a brand-new install currently.

File: data_management/management/commands/add_example_data.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    def handle(self, **options):
+
+        force = options["force"]
+
+        try:
+            user = get_user_model().objects.all()
+            group = Group.objects.all()
+            objects = Object.objects.all()
+
+            if (len(user) > 0 or len(group) > 0 or len(objects) > 0) and not force:
```
Comment (nathan-cummings-ukaea) [inline data_management/management/commands/add_example_data.py:34]: Admin user is automatically created so `len(user)` will always be greater than 0

**Comment by nathan-cummings-ukaea** [comment]
Yep, that works. Cheers @antony-wilson

**Comment by agbeltran** [comment]
It seems that this is ready to be merged? @antony-wilson @nathan-cummings-ukaea

**Review by nathan-cummings-ukaea — APPROVED** [review]

**Pull request merged**